cmake/config/linux-gcc.txt

```diff
@@ -309,7 +309,7 @@ macro (add_copy_py_file_to_sandbox_py_module_command py_src module_name)
     get_relative_path_from_module_name (relative_module_path ${module_name})
     set(module_path "${py_path}/${relative_module_path}")
 
-    add_custom_command (TARGET appleseed.python POST_BUILD
+    add_custom_command (TARGET appleseed.python.copy_py_files POST_BUILD
```

Comment:
you need to make the same changes in cmake/config/windows.txt and the one for OSX too.